### DISCUSSION TITLE
Usage Question / Crediting

### DISCUSSION THREAD

**Discussion by klinbee** [jacobsjo/snowcapped#30] [Q&A] [answered]
Hello! 

I've recently discovered your world generation tools and they are very amazing and very useful! I was wondering though what specifically I would need to do in terms of crediting your tool for creating a Data-pack with it, if I were to then upload that Data-pack. I tried reading over the licensing, but legal jargon is difficult to understand...

Just making sure before I upload any Data-packs. Thank you for your time and your hard work! It's incredibly underrated.

BTW, if this wasn't the correct area to go for this, I apologize. I wanted to email or DM you, but I couldn't find any contact information after looking for a few minutes, so I decided here was easiest.

**Comment by jacobsjo** [comment, marked as answer]
Hi!

As with most tools, simply using the tools doesn't require any attribution at all. Attribution is only required if you were to redistribute the tools themselves.

Of course I'd still appreciate an attribution, but that is entirely up to you.

I'm glad you find my tools useful, thanks for the complements!

**Reply by klinbee** [reply]
Alright sounds good! I'll definitely give a shoutout anyways on everything I end up making that uses it because it is so useful, and I think that more people should know about it. I also believe that you should always give credit to others!
